indexer/packages/kafka/src/constants.ts

```diff
@@ -2,6 +2,6 @@ export const TO_ENDER_TOPIC: string = 'to-ender';
 
 export const ORDERBOOKS_WEBSOCKET_MESSAGE_VERSION: string = '1.0.0';
 export const SUBACCOUNTS_WEBSOCKET_MESSAGE_VERSION: string = '2.4.0';
-export const TRADES_WEBSOCKET_MESSAGE_VERSION: string = '1.1.0';
+export const TRADES_WEBSOCKET_MESSAGE_VERSION: string = '1.2.0';
```

Comment:
Major version change? Some properties were removed and replaced with a different type of property.